### DISCUSSION TITLE
Scrollbar dissapears when maximizing a view

### DISCUSSION THREAD

**Discussion by Carlos-mb** [Security-Onion-Solutions/securityonion#13614] [2.4]
### Version

2.4.90

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

other (please provide detail below)

### Hardware Specs

Exceeds minimum requirements

### CPU

12

### RAM

52

### Storage for /

2TB

### Storage for /nsm

2TB

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi all

I am experiencing that when I maximize a view that has many rows, there is no scroll bar on the new page and it is not possible to see the entire table.

My idea was to have a single page with all rows, maybe 70 rows in this case, in order to copy it for a report, but I have to set the "Rows per page" in less than 15 and use the arrows to copy separated pages content to paste it in the report. 

I have tried it in 2 different SO installations and 2 browsers (Chrome and Firefox)

Is this an issue?

Regards,
Carlos

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by Carlos-mb** [comment]
When I press "back" in browser there is not scroll bar, I have to refresh the page to make it appears.

**Comment by dougburks** [comment]
Are you using Kibana dashboards or SOC dashboards? It sounds like you're using Kibana. If that's the case, then you might try updating to Security Onion 2.4.100 which includes a newer version of Kibana to see if the issue persists there. If so, then you might report this issue to the Kibana team (we don't develop Kibana, we simply include it in the platform).
